### DISCUSSION TITLE
Is it possible to get the actual verbatim ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [openai/whisper#712] [Q&A] [answered]
Hi,

When I use Whisper with the stock parameters the transcriptions are always clean, ie  without "uh", "erm", repetitions, etc... This is good because it makes awesome transcripts.

But if I really want exactly what's been said like in "Uh, I don't know, uh, maybe 100, 153, or or even more". Now Whisper outputs "I don't know, maybe 153 or even more", and I wonder if there is an option to get the actual sentence with all that noisy words ?

Thanks in advance!

**Comment by jongwook** [comment, marked as answer]
This is somewhat steerable using prompts, such as https://github.com/openai/whisper/discussions/625#discussioncomment-4319892

**Reply by Ca-ressemble-a-du-fake** [reply]
Oh that's good, gonna try this, thank you!

**Reply by Ca-ressemble-a-du-fake** [reply]
That's a very sensitive way to steer whisper:laughing:! If I use "euh, je, je, j'ai, euh, y a, y a, le, le" it will find an initial "euh" but if I use "euh, je, je **pense que, euh**, j'ai, euh, y a, y a, le, le" (in bold what I added) to get the other "euh"s in the middle of the speech, then no "euh"s are found (neither starting, nor in the middle).

**Comment by LaurinmyReha** [comment]
Please check out this Whisper variant which was finetuned specifically with filler detection and verbatim transcription in mind:

https://github.com/nyrahealth/CrisperWhisper

**Reply by jrmz2k** [reply]
Hi LaurinmyReha,

Do you know if this one can be run through Google Colab just like the regular Whisper?

Thanks in advance!

**Reply by LaurinmyReha** [reply]
Sure. If you are using the transformers library for example it should be completely compatible with the transformers implementation. There is a code snippet of how to use it in the repo.
